src/test/java/com/dsl/clauses/linking/ClauseBuilderTest.java

```diff
@@ -282,8 +282,8 @@ void returnsWithWrongType() {
     @Test
     void createMultiplePaths() {
         String s = create()
-            .exp(node("n:Name").props("name", "'Fred'"))
-            .exp("(s:Colour {colour: 'orange'})")
+            .createPath(node("n:Name").props("name", "'Fred'"))
```

Comment:
`String s = createPath.append().append()`
Doesn't it look better?

Reply:
yes, the problem is that the merge/with/create/match have the same logic, and the name cant be the same or it will not work out

Reply:
Ill try to make it better, let me see if it can be done